TESTS/host_tests/usb_device_serial.py

```diff
@@ -0,0 +1,346 @@
+"""
+mbed SDK
+Copyright (c) 2018 ARM Limited
```

Comment:
Please add SPDX identifiers for new files

Reply:
done